rcl_interfaces/msg/LoggerLevel.msg

```diff
@@ -0,0 +1,13 @@
+# All available logger levels
+uint8 LOG_LEVEL_UNKNOWN = 0
+uint8 LOG_LEVEL_DEBUG = 10
+uint8 LOG_LEVEL_INFO = 20
+uint8 LOG_LEVEL_WARN = 30
+uint8 LOG_LEVEL_ERROR = 40
+uint8 LOG_LEVEL_FATAL = 50
+
+# The logger name.
+string name
+
+# The logger level
+uint8 level
```

Comment:
Doesn't this need to be the same size as what rcutils can represent?  In https://github.com/ros2/rcutils/blob/04aa9804feb46403f0058f4b089134a6985e19d3/include/rcutils/logging.h#L171-L179 , it is an enum, which I think is at least 4 bytes (uint32).

```suggestion
uint32 level
```

Reply:
Updated